e2e/playwright/search/src/tests/search-filters-tags.e2e.ts

```diff
@@ -0,0 +1,92 @@
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, Utils, test, NodesApi, TrashcanApi, TagsApi, FileActionsApi } from '@alfresco/playwright-shared';
+import { TagEntry, TagPaging } from '@alfresco/js-api';
+
+test.describe('Search - Filters - Tags', () => {
+  let nodesApi: NodesApi;
+  let trashcanApi: TrashcanApi;
+  let tagsApiAdmin: TagsApi;
+  let fileActionsApi: FileActionsApi;
+  let file1Id: string;
+  let file2Id: string;
+  let tagEntries: TagEntry[];
+
+  const random = Utils.random();
+  const username = `user1-${random}`;
+  const file1 = `${random}-file1`;
+  const file2 = `${random}-file2`;
+  const tags = [{ tag: `${random}-tag1` }, { tag: `${random}-tag2` }];
+
+  test.beforeEach(async ({ loginPage }) => {
+    await Utils.tryLoginUser(loginPage, username, username, 'beforeEach failed');
+  });
+
+  test.beforeAll(async () => {
+    try {
+      const apiClientFactory = new ApiClientFactory();
+      await apiClientFactory.setUpAcaBackend('admin');
+      await apiClientFactory.createUser({ username });
+      nodesApi = await NodesApi.initialize(username, username);
+      trashcanApi = await TrashcanApi.initialize(username, username);
+      tagsApiAdmin = await TagsApi.initialize('admin');
+      fileActionsApi = await FileActionsApi.initialize(username, username);
+      const node1 = await nodesApi.createFile(file1);
+      file1Id = node1.entry.id;
+      const node2 = await nodesApi.createFile(file2);
+      file2Id = node2.entry.id;
+      await fileActionsApi.waitForNodes(file1, { expect: 1 });
+      await fileActionsApi.waitForNodes(file2, { expect: 1 });
+      tagEntries = await tagsApiAdmin.createTags(tags);
+      await tagsApiAdmin.assignTagToNode(file1Id, tags[0]);
+      await tagsApiAdmin.assignTagToNode(file2Id, tags[1]);
+    } catch (error) {
+      if (tagEntries == null) {
+        console.error(`beforeAll failed null : ${error}`);
+      }
+      console.error(`beforeAll failed: ${error}`);
+    }
+  });
+
+  test.afterAll(async () => {
+    await Utils.deleteNodesSitesEmptyTrashcan(nodesApi, trashcanApi, 'afterAll failed');
+    await tagsApiAdmin.deleteTag(`${(tagEntries as TagPaging).list.entries[0].entry.id}`);
```

Comment:
To not repeat yourself you could do it in a loop iterating over `tagEntries` also `${(tagEntries as TagPaging).list.entries[0].entry.id}` doesn't look correct you can simply use `tagEntry.entry.id` inside the loop

Reply:
discussed and will be corrected when https://hyland.atlassian.net/browse/ACS-7706 will be done